scripts/build

```diff
@@ -6,13 +6,22 @@ require('./util/work-from-root')();
 
 var cleanDirectory = require('./util/clean-directory');
 var buildProject = require('./util/build-project');
+var docBuilder = require('./util/build-doc')({ docsBranchName: 'gh-pages-6', commitDocs: true, rmSrcDir: true });
```

Comment:
That has been removed now. I created multiple branches while performing tests locally.